core/templates/pages/topic-editor-page/services/create-new-skill-modal.service.spec.ts

```diff
@@ -122,19 +122,24 @@ describe('Create New Skill Modal Service', () => {
   beforeEach(() => {
     createNewSkillModalService = TestBed.inject(CreateNewSkillModalService);
     alertsService = TestBed.inject(AlertsService);
-    alertsService = (alertsService as unknown) as
-      jasmine.SpyObj<AlertsService>;
+    alertsService = alertsService as jasmine.SpyObj<AlertsService>;
     imageLocalStorageService = TestBed.inject(ImageLocalStorageService);
-    imageLocalStorageService = (imageLocalStorageService as unknown) as
+    imageLocalStorageService = imageLocalStorageService as
       jasmine.SpyObj<ImageLocalStorageService>;
     topicsAndSkillsDashboardBackendApiService =
     TestBed.inject(TopicsAndSkillsDashboardBackendApiService);
-    topicsAndSkillsDashboardBackendApiService =
-    (topicsAndSkillsDashboardBackendApiService as unknown) as jasmine
-    .SpyObj<TopicsAndSkillsDashboardBackendApiService>;
-    mockSkillCreationBackendApiService =
-    (TestBed.inject(SkillCreationBackendApiService) as unknown) as
-    MockSkillCreationBackendApiService;
+    topicsAndSkillsDashboardBackendApiService = (
+      topicsAndSkillsDashboardBackendApiService as
+      jasmine.SpyObj<TopicsAndSkillsDashboardBackendApiService>
+    );
+    mockSkillCreationBackendApiService = (
+      // This throws "Type 'MockSkillCreationBackendApiService' is not
+      // assignable to type desire". We need to suppress this error because of
+      // the need to test validations. This happens because the
+      // MockSkillCreationBackendApiService is a class and not an interface.
+      // @ts-ignore
+      TestBed.inject(SkillCreationBackendApiService) as
+    MockSkillCreationBackendApiService);
```

Comment:
```suggestion
    mockSkillCreationBackendApiService = (
      // This throws "Type 'MockSkillCreationBackendApiService' is not
      // assignable to type desire". We need to suppress this error because of
      // the need to test validations. This happens because the
      // MockSkillCreationBackendApiService is a class and not an interface.
      // @ts-ignore
      TestBed.inject(SkillCreationBackendApiService) as
      MockSkillCreationBackendApiService
    );
```

Reply:
Done